src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/IModelBindingContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    public interface IModelBindingContext
```

Comment:
Why add this interface? It seems to cause a lot of additional churn, there's one implementation, and no extensibility point here.